tests/50federation/30room-join.pl

```diff
@@ -52,103 +52,118 @@ sub assert_is_valid_pdu {
 }
 push our @EXPORT, qw( assert_is_valid_pdu );
 
-test "Outbound federation can send room-join requests",
-   requires => [ local_user_fixture(), $main::INBOUND_SERVER,
-                 federation_user_id_fixture() ],
+foreach my $versionprefix ( qw( v1 v2 ) ) {
+   test "Inbound federation supports $versionprefix /send_join",
```

Comment:
wait. what is this testing?

Reply:
That a client requesting to join a remote room results in a v1 `make_join` request and a v[12] `send_join` request, and that the server can process the responses. I just realised I mixed up "inbound" and "outbound" in the test names, I've fixed that in https://github.com/matrix-org/sytest/pull/760/commits/78f8a530f6e09a78eda19e12f8a5c3e2c917cb9e